Orange/canvas/application/errorreporting.py

```diff
@@ -189,7 +193,10 @@ def _find_last_frame(tb):
             err_module = '{}:{}'.format(
                 frame.tb_frame.f_globals.get('__name__', frame.tb_frame.f_code.co_filename),
                 frame.tb_lineno)
-            err_locals = pformat(OrderedDict(sorted(frame.tb_frame.f_locals.items())))
+            try:
+                err_locals = pformat(OrderedDict(sorted(frame.tb_frame.f_locals.items())))
+            except RuntimeError:  # RecursionError is not supported by Python < 3.5
```

Comment:
I assume RecursionError here is triggered because _something_ in the above line triggers an exception which unhandled gets passed to our ErrorReporting hook, and the thing reiterates. Could we instead try to catch that something?

Reply:
Added the link despite you've already found it.

Reply:
RecursionError is a bit deep. Just catch Exception ...
```py
from Orange.util import try_

err_locals = OrderedDict(sorted(frame.tb_frame.f_locals.items()))
err_locals = try_(lambda: pformat(err_locals), try_(lambda: str(err_locals)))
```